### PR TITLE
Add calculator operation

### PR DESCRIPTION
**Pull request by makostanyan**

### CODE REVIEW

File: src/test/java/CalculatorTest.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class CalculatorTest {
+
+    Calculator calculator = new Calculator();
```
Comment (chandranik90) [inline src/test/java/CalculatorTest.java:6]: Always specify the access modifier, if there is no specific reason for not doing it.

**Review by chandranik90 — APPROVED** [review]

File: src/test/java/CalculatorTest.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        calculator.divide(50,0);
+    }
+
+    @Test
+    public void multiplicationTest(){
+        double actualMultiplication = calculator.multiplication(13,2.5);
+        double expectedMultiplication = 32.5;
+        Assert.assertEquals(actualMultiplication,expectedMultiplication);
+    }
+
```
Comment (testforham) [inline src/test/java/CalculatorTest.java:47]: unnecessary blank lines

File: src/test/java/CalculatorTest.java [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    @Test
+    public void subtractionTest(){
+        double actualSubtraction = calculator.subtraction(13,2.5);
+        double expectedSubtraction = 10.5;
+        Assert.assertEquals(actualSubtraction,expectedSubtraction);
+    }
+
+    @Test
+    public void subtractionBooleanTest(){
```
Comment (testforham) [inline src/test/java/CalculatorTest.java:23]: What are the differences between this and `subtractionTest()` method?

**Review by testforham — COMMENTED** [review]
Please reformat the code before pushing it. (`Ctrl+Alt+L` in case of Windows machine, inside the `.class` file)

**Pull request merged**